Tevling/Pages/Statistics.razor.cs

```diff
@@ -228,7 +245,81 @@ await _module.InvokeVoidAsync(
                 await _module.InvokeVoidAsync(
                     "drawChart",
                     Durations.ToDictionary(stat => stat.Type, stat => stat.LastTimePeriodAggregate),
-                    months,
+                    timePeriodArray,
+                    "TheChart",
+                    Loc["TotalTime"] + " [h]",
+                    "h");
+                break;
+            default:
+                throw new Exception("Unknown challenge measurement");
+        }
+    }
+
+    private async Task DrawChartMobile(string[] timePeriodArray)
```

Comment:
It looks like this method is equal to `DrawChartDesktop` with the exception of calculating the totals. Perhaps only that part could be extracted into separate methods, and the bulk of these methods could be re-used for both?